app/data/markdown-renderer.ts

```diff
@@ -18,15 +19,19 @@ export const makeMD = (opts: {
 
   // Headings
   const headingSizes = {
-    '1': 'text-xl font-bold mb-3',
-    '2': 'text-xl font-semibold mb-3',
-    '3': 'text-xl font-semibold mb-3',
-    '4': 'text-xl font-semibold mb-3',
-    '5': 'text-lg font-medium mb-2',
-    '6': 'text-base font-medium mb-1',
```

Comment:
Why did we drop the text size classes?

Reply:
@MarshallOfSound some of the headings in the PR details have `text-lg` set to them so I wanted to make the heading level appearances more consistent:

https://github.com/electron/release-status/blob/566e0faa977f95bffc929e61344aa66e1a431c48/app/routes/pr/details.tsx#L263-L266

With the PR in the state it is now, I don't actually think we _need_ this change necessarily (headings generated by `markdown-renderer` in the compare view are `h5` after https://github.com/electron/release-status/pull/167/commits/9eb23f3466658a2693381f6af6bb37f29a0a8d34), but we could instead do the following to match the non-MD heading styles:

```
h1: text-xl
h2: text-xl
h3: text-lg
h4: text-lg
h5: text-base
h6: text-base
```

Reply:
https://github.com/electron/release-status/pull/167/commits/457021e72a97b18c5fda5b009129bf029b8e96dd